CHANGELOG.md

```diff
@@ -1,6 +1,33 @@
 Change Log / Release Log for mallocMC
 ================================================================
 
+2.5.0crp
+--------
+**Date:** 2021-02-XX
+
+This release removes the native usage of CUDA by alpaka.
+Attention: This release depends on an unreleased [alpaka 0.5.0dev](https://github.com/alpaka-group/alpaka/commit/34870a73ecf702069465aa030fbdf301c4d22c61) 
```

Comment:
I guess we should do another release then based on alpaka 0.6.0.

Reply:
yes